site-src/geps/gep-1619.md

```diff
@@ -0,0 +1,89 @@
+# GEP-1619: Session Persistence and Session Affinity
+
+* Issue: [#1619](https://github.com/kubernetes-sigs/gateway-api/issues/1619)
+* Status: Provisional
+
+(See definitions in [GEP Status][/contributing/gep#status].)
+
+## TLDR
+
+This GEP proposes a definition of session persistence and session affinity as the foundation of future session persistence related improvements. This GEP also outlines the ways session persistence could be achieved by implementations.
+
+## Goals
+- Define session persistence and session affinity to establish a common language
+- Identify differences in session persistence functionality between implementations
+
+## Non-Goals
+- Create proposals for adding session persistence configuration to Gateway API object specification
```

Comment:
Although this may be a non-goal for this particular "provisional" stage of the GEP, I think it would need to be a goal of the GEP as a whole. Or maybe I'm misunderstanding the purpose of writing this up.

Reply:
Agreed, it seems that this should be an _eventual_ goal of this KEP, however it's reasonable for the first iteration to just be the "what and the why" without the how and still merge.

Reply:
```suggestion
```